docs/source/testing/unit.rst

```diff
@@ -18,4 +18,148 @@
 Writing Unit Tests for NWChemEx
 ###############################
 
-TODO: Write me!!!!
+Within the first party NWX libraries, we aim for extensive unit testing to 
```

Comment:
NWChemEx.

Reply:
I agree with changing this occurrence of "NWX," since it is the only usage in this document.